### PR TITLE
Manually send events

### PR DESCRIPTION
**Pull request by mjoss17**

### CODE REVIEW

**Comment by rnathuji** [comment]
@mjoss17 - I think it's fine that you're using this codebase as a harness to test the spike code, but I don't think we want to merge any of this into `main` since this repo is production code vs spike code (similar to how later we don't intend to merge changes to `k12-apps-raise` into `main` there). 

That being said, it's probably still useful to push code to a branch and have it reviewed as part of the spike. I created a dedicated `k12-173/spike-code` branch in this repo and switched this PR to merge against that branch accordingly.

File: moodlecli/main.py [hunk 1]
```diff
@@ -287,3 +287,10 @@ def export_bulk_csv(output_csv):
         utils.bulk_export_csv_course_ids()
     )
     writer.writeheader()
+
+
+@cli.command()
+@click.argument('content_id')
+def log_event_manually(content_id):
+    moodle = get_moodle_client()
```
Comment (rnathuji) [inline moodlecli/main.py:295]: In so much as this is a helper command to test the APIs for the spike, let's create a single command `trigger-contentloaded-event` and have it take a parameter like `type` which would be a `click.Choice`. In this PR we can just allow a single option of `direct`, but then later we can add `moodle`. That way it's set up for us to use it to test both endpoints for k12-182 and k12-183.

Comment (mjoss17) [reply moodlecli/main.py:295]: Oh that's handy. Sounds good 👍

Comment (rnathuji) [reply moodlecli/main.py:299]: Ah, sorry, I may have created some confusion here in my initial comment. I think including the `content_id` as an input is still a good idea. I think you may have made a change to hard code that now to `test_id`. Similarly, I think you had the better call signature for the moodle client function previously as the convention is we create the data structure expected by the service there versus in this CLI handler.

**Review by rnathuji — APPROVED** [review]

**Pull request merged**